WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -481,19 +484,22 @@ private extension NewOrderViewModel {
 
                 let shippingMethodTitle = order.shippingLines.first?.methodTitle ?? ""
 
+                let feesBaseAmountForPercentage = itemsTotal.adding(shippingTotal)
```

Comment:
Nit: Should we have a comment here (or somewhere) about what should be included in this base amount? I'm just thinking about when we add the tax line, since to match Core behavior we need to make sure it's included in this amount, too.

Reply:
We have task https://github.com/woocommerce/woocommerce-ios/issues/6236 that should take care of this logic. I'll add TODO comment.